samples/template-operator/Makefile

```diff
@@ -103,16 +104,7 @@ $(LOCALBIN):
 
 ########## Kyma CLI ###########
 KYMA_CLI_CHECKOUT_PATH = "/tmp/operator/make-build/kyma-cli"
-KYMA ?= $(LOCALBIN)/kyma
-.PHONY: get-kyma
-get-kyma:
-	if [ ! -d $(KYMA_CLI_CHECKOUT_PATH) ]; then GOBIN=$(LOCALBIN) git clone https://github.com/kyma-project/cli $(KYMA_CLI_CHECKOUT_PATH) -b main --depth 100; fi
-	cd $(KYMA_CLI_CHECKOUT_PATH) && git checkout $(KYMA_VERSION)
-
-.PHONY: kyma
-kyma: get-kyma $(KYMA) ## Download & Build Kyma CLI locally if necessary.
-$(KYMA): $(LOCALBIN)
-	cd $(KYMA_CLI_CHECKOUT_PATH) && make resolve && go build -o $(KYMA) -ldflags '-s -w -X github.com/kyma-project/cli/cmd/kyma/version.Version=$(KYMA_VERSION)' ./cmd 
+KYMA ?= /Users/D067928/SAPDevelop/go/src/github.com/kyma-project/cli/bin/kyma-darwin
```

Comment:
Since it's hard to predict where is the `kyma` located in the user's machine, perhaps we just fail-fast if `kyma-darwin` is not found with a message: "Please export an environment variable  KYMA providing a location of kyma-darwin binary location"

Reply:
Ill make it so that we download it dynamically for now and then we can store it in local-bin. We can optimize later on if its not found

Reply:
👍